lib/action_cable/railtie.rb

```diff
@@ -0,0 +1,19 @@
+module ActionCable
+  class Railtie < Rails::Railtie
+    config.action_cable = ActiveSupport::OrderedOptions.new
+
+    initializer "action_cable.logger" do
+      ActiveSupport.on_load(:action_cable) { self.logger ||= ::Rails.logger }
+    end
+
+    initializer "action_cable.set_configs" do |app|
+      options = app.config.action_cable
+
+      options.allowed_request_origins ||= "http://localhost:3000" if ::Rails.env.development?
```

Comment:
@kaspath When the value is used at https://github.com/rails/actioncable/blob/master/lib/action_cable/connection/base.rb#L178 it is converted into an array.